hectorsector-orlando.md

```diff
@@ -0,0 +1,10 @@
+# Orlando, FL
+
+### Restaurants
+
+- The Pharmacy
+- Rocco's Tacos
+
+### Attractions
+
+- Disney World!
```

Comment:
@githubteacher this place is kind of overpriced, can you please recommend something on the cheap? :money_with_wings: